### PR TITLE
Move bank equipment button into title bar

### PR DESCRIPTION
**Pull request by raiyni**
java_PcNeH5gDiX

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java [hunk 1]
```diff
@@ -73,7 +73,10 @@
 import net.runelite.api.widgets.JavaScriptCallback;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetConfig;
+import static net.runelite.api.widgets.WidgetID.BANK_GROUP_ID;
```
Comment (abextm) [inline runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java]: don't static import these

**Comment by dekvall** [comment]
The original equipment button doesn't get restored until you open a new bank window if you toggle the tag-tab setting or the plugin entirely.

**Comment by raiyni** [comment]
> The original equipment button doesn't get restored until you open a new bank window if you toggle the tag-tab setting or the plugin entirely.

I'm not sure that really matters

File: runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java (hunk 1 above)
Comment (abextm) [reply runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java]: this still applies

**Comment by Adam-** [comment]
I don't really have an opinion on this

**Comment by raiyni** [comment]
> These widget IDs have changed, so this PR is not working anymore.

fixed widget ids

**Comment by deathbeam** [comment]
Needs rebase

**Pull request merged**